src/components/biggive-branded-image/biggive-branded-image.tsx

```diff
@@ -41,13 +41,17 @@ export class BiggiveBrandedImage {
    */
   @Prop() charityUrl: string;
 
+  /**
+   * This is only used as a background, presentational "bonus" so we are setting
+   * `role="presentation"` every time instead of accepting alt text.
+   */
```

Comment:
I'm not sure it's true to say that its always background or presentational. For donate-frontend the only place we use this is in the campaign detail page, where it shows the image at `campaign.bannerUri`. This often has text and/or meaningful content images. Only a small part of the image is covered up by the charity logo.

e.g. at https://donate.biggive.org/campaign/a05WS0000027gZFYAY

![image](https://github.com/user-attachments/assets/61cb0c9c-e9bd-45dd-ad27-d4de042d6f12)